python_modules/dagster/dagster/_config/errors.py

```diff
@@ -25,42 +23,42 @@ class DagsterEvaluationErrorReason(Enum):
     FIELD_ALIAS_COLLISION = "FIELD_ALIAS_COLLISION"
 
 
-@dagster_model
+@check.record
```

Comment:
I think this should be just `@record` personally.

Reply:
ya i think just using `@record` is cleaner, but given how `check` tends to get imported I think this will happen unless we like don't export it at the top level of check and you have to grab it else where. Do you think its worth doing that? Just try to establish a common pattern of not doing `check.record` ? Something else?